adapters/jwplayer/utils.go

```diff
@@ -0,0 +1,199 @@
+package jwplayer
+
+import (
+	"encoding/json"
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/macros"
+	"net/url"
+	"strconv"
+	"strings"
+	"text/template"
+)
+
+func ParseExtraInfo(v string) ExtraInfo {
+	var extraInfo ExtraInfo
+	if err := json.Unmarshal([]byte(v), &extraInfo); err != nil {
+		extraInfo = ExtraInfo{}
+	}
+
+	if extraInfo.TargetingEndpoint == "" {
+		extraInfo.TargetingEndpoint = "https://content-targeting-api.longtailvideo.com/property/{{.SiteId}}/content_segments?content_url=%{{.MediaUrl}}&title={{.Title}}&description={{.Description}}"
```

Comment:
are there config files we can define this in rather than hard-coding it in this class?

Reply:
This is a fallback. The url is defined in the config, see https://github.com/jwplayer/prebid-server/pull/17/files#diff-fe44f09c4d5977b5f5eaea29170b6a0748819c9d02271746a20d81a5f3efca17R812